.eslintrc.js

```diff
@@ -36,5 +36,21 @@ module.exports = {
       files: ['**/.eslintrc.js'],
       env: {node: true},
     },
+    {
+      files: './packages/checkout-ui-extensions/docs/staticPages/examples/**',
+      rules: {
+        'no-console': 'off',
+        'react/react-in-jsx-scope': 'off',
+        '@shopify/jsx-no-hardcoded-content': 'off',
+        'import/no-extraneous-dependencies': 'off',
+        'prettier/prettier': [
+          'error',
+          {
+            printWidth: 50,
+          },
+        ],
+        'promise/catch-or-return': 'off',
+      },
+    },
```

Comment:
Hmm, this will be really hard to keep in sync with `checkout-web`. I had this idea: https://github.com/Shopify/checkout-web/pull/18336#issuecomment-1442522729